src/backend/dx12/Cargo.toml

```diff
@@ -19,16 +19,11 @@ name = "gfx_backend_dx12"
 
 [dependencies]
 bitflags = "1.0"
-log = "0.3"
+derivative = "1"
 gfx-hal = { path = "../../hal", version = "0.1" }
-d3d12-sys = { git = "https://github.com/msiglreith/winapi-rs.git", branch = "gfx" }
-d3dcompiler-sys = { git = "https://github.com/msiglreith/winapi-rs.git", branch = "gfx" }
-dxgi-sys = { git = "https://github.com/msiglreith/winapi-rs.git", branch = "gfx" }
-dxguid-sys = { git = "https://github.com/msiglreith/winapi-rs.git", branch = "gfx" }
-kernel32-sys = { git = "https://github.com/msiglreith/winapi-rs.git", branch = "gfx" }
+log = "0.3"
+smallvec = "0.4"
 spirv_cross = "0.4.2"
-user32-sys = { git = "https://github.com/msiglreith/winapi-rs.git", branch = "gfx" }
-winapi = { git = "https://github.com/msiglreith/winapi-rs.git", branch = "gfx" }
+winapi = { version = "0.3", features = ["basetsd","d3d12","d3d12sdklayers","d3d12shader","d3dcommon","d3dcompiler","dxgi1_2","dxgi1_3","dxgi1_4","dxgiformat","dxgitype","minwindef","synchapi","unknwnbase","winbase","windef","winerror","winnt","winuser"] }
 winit = { version = "0.7", optional = true }
 wio = { git = "https://github.com/msiglreith/wio-rs.git", branch = "stable" }
```

Comment:
would this be the last thing in github dependencies?

Reply:
yes, iirc
retep998 mentioned to work on it soon™️ !